woocommerce/google-listings-and-ads
# Set Node/npm Engines

### Changes proposed in this Pull Request:

- Set the strict-engines directive in `.npmrc `
- Set the engines in `package.json`


### Detailed test instructions:


1.   Run `nvm use 14` and see the npm compilation builds
3.  Run `nvm use 18` (or any other version, not 14)  and see the npm compilation fails


### Additional details:


### Changelog entry

> Dev - Set engines for the repository
